Pages/HomePage/style.css

```
@@ -30,11 +30,16 @@
 
 /* Media query para celulares */
 body {
-    font-family: 'Montserrat', sans-serif;
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    font-family: "Montserrat", sans-serif;
     background-color: var(--white);
     min-height: 100vh;
-    width: 100%;
-    margin-bottom: 100px;
+
+    width: 100vw;
```

isso não deve ser alterado, voltar para
```suggestion
    width: 100%;
    margin-bottom: 100px;
```